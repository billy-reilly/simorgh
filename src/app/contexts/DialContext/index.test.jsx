import React from 'react';
import { shouldMatchSnapshot } from '../../helpers/tests/testHelpers';
import { ComponentUsingContext } from '../../helpers/tests/mockComponents';
import { DialContext, DialContextProvider } from './index';

describe('DialContext', () => {
  const dials = { dial: 'value' };

  shouldMatchSnapshot(
    `should provide a dials object`,
    <DialContextProvider dials={dials}>
      <ComponentUsingContext context={DialContext} />
    </DialContextProvider>,
  );
});
