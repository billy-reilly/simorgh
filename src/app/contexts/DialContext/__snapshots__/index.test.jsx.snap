// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DialContext should provide a dials object 1`] = `
<span>
  {"dial":"value"}
</span>
`;
